next/src/components/_global/faq/faq.tsx

```diff
@@ -38,25 +26,26 @@ export default function Faq({ data: { centralizedHeading, description, faq, imag
             data={image}
           />
           <div className={styles.items}>
-            {faq.map(({ title, description, image }, i) => (
-              <div
-                className={i == activeIndex ? `${styles.item} ${styles.active}` : styles.item}
-                key={i}
-              >
+            {faq.map(({ title, description, image }, i) => {
+              const arrow = <Arrow />;
+              const Image = (
                 <Img
                   className={styles.image}
                   data={image}
                 />
-                <Markdown.h3 className={styles.heading}>{title}</Markdown.h3>
-                <button
-                  className={styles.icon}
-                  onClick={() => handleButtonClick(i)}
-                >
-                  <Arrow />
-                </button>
-                <Markdown className={styles.description}>{description}</Markdown>
-              </div>
-            ))}
+              );
+
+              const Heading = <Markdown.h3 className={styles.heading}>{title}</Markdown.h3>;
+
+              const Description = <Markdown className={styles.description}>{description}</Markdown>;
+
+              return (
+                <FaqItem
+                  data={{ Image, Heading, Description, arrow, key: i }}
+                  key={i}
+                />
```

Comment:
FAQ should be rendered semantically as a `<details>` and `<summary>` tags. Change it globally. Moreover whole tab (with title) not only icon should be clickable.
1. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details